ui-tests/cypress/support/pageObjects/app/AppTour.js

```diff
@@ -43,10 +43,7 @@ export default class AppTour extends WebElement {
   }
 
   close() {
-    //TODO find a better way to identify when the page loaded
-    cy.wait(1000); // Wait until the page loads
-    //TODO find a better way to identify when the dialog animation is completed
-    cy.wait(2000); // Wait until the animation of the App Tour dialog is completed
+    cy.wait(1000);
```

Comment:
Do we still need this wait?

Reply:
it did left a comment saying it's for waiting for page load. I'll try it to remove it and test it out if we do really need it or not

Reply:
cypress fails to click the modal if we remove that `cy.wait`. Looks like we'll retain that @g-di-martino

Reply:
ok; please, leave the comment on top of it as well